web/modules/custom/ai_screening/src/Drush/Commands/Commands.php

```diff
@@ -117,15 +117,15 @@ public function accessCheck(
   /**
    * Pre command hook for pm:install.
    */
-  #[CLI\Hook(HookManager::PRE_COMMAND_HOOK, target: '*')]
+  #[CLI\Hook(HookManager::PRE_COMMAND_HOOK, target: 'pm:install')]
   public function preCommand(CommandData $commandData): void {
     $this->accountSwitcher->switchTo(new UserSession(['uid' => 1]));
   }
 
   /**
    * Post command hook for pm:install.
    */
-  #[CLI\Hook(HookManager::POST_COMMAND_HOOK, target: '*')]
+  #[CLI\Hook(HookManager::POST_COMMAND_HOOK, target: 'pm:install')]
```

Comment:
Is this a fix for the runtime exception “No more accounts to revert to.” (cf. https://github.com/drupal/drupal/blob/b9a10a16d0f7188e789ccaf755f28377bc343c39/core/lib/Drupal/Core/Session/AccountSwitcher.php#L79)? I've just run into this …

May wrapping the call to `switchBack` in a try-catch block is a ~better~ solution, i.e. just ignoring the exception.

Reply:
Might be, don't quite remember. But i got an error when trying to use drush:user-login, and i'm not sure what other drush commands might not work with that so i just reverted. I don't think the change to * i originally added makes any difference